avocado/utils/network/interfaces.py

```diff
@@ -75,6 +75,14 @@ def _get_interface_details(self, version=4):
             log.error(msg)
             raise NWException(msg)
 
+    def _get_bondinterface_details(self):
+        cmd = "cat /sys/class/net/{}/bonding/mode /sys/class/net/{}/bonding/slaves".format(self.name, self.name)
+        try:
+            output = run_command(cmd, self.host)
+            return output
+        except Exception as ex:
+            raise NWException("Bond interface details not found")
```

Comment:
You raise the exception as "ex" and never use this variable.

Reply:
Would be nice instead of hiding the exception to exporting it to upper layers, since this is an generic exception and could get another problems.

```python
except Exception as ex:
   raise NWException("Bond interface details not found: {}".format(str(ex))
```